File: 9-palindrome-number/9-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
            unsigned long long int sum = 0,copy = x;
            int b;
            while(copy>0){
                b = copy % 10;
                sum = sum *10 + b;
                copy = copy/10;
            }
            return sum == x;
    }
};

File: 9-palindrome-number/9-palindrome-number.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function(x) {
    if(x < 0){
    return false
    }else if (x <=9){
        return true;
    }else{
        let sum = 0, b, a = x;
        while(a>0){
            b = a % 10;
            sum = sum*10 + b;
            a = parseInt(a /10);
        }
        console.log("sum", sum)
        return sum === x;
        
    }
    
};